src/components/movie-card/movie-card.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./movie-card.css";
+import { imageUrl } from "../../services/instances";
+import { Flex, Heading, Icon, Text } from "@chakra-ui/react";
+import { StarIcon } from '@chakra-ui/icons'
+
+
+const MovieCard = (props) => {
```

Comment:
Let's define this component in the above folder instead of having a separate one for `movie-card`.

So it should be here: `/src/components/movie-card.js`.